package/MDAnalysis/coordinates/LAMMPS.py

```diff
@@ -620,19 +661,47 @@ def _read_next_timestep(self):
             except IndexError:
                 raise ValueError("No coordinate information detected")
         elif not self.lammps_coordinate_convention in convention_to_col_ix:
-            raise ValueError(f"No coordinates following convention {self.lammps_coordinate_convention} found in timestep")
+            raise ValueError(
+                f"No coordinates following convention {self.lammps_coordinate_convention}"\
```

Comment:
No need for explicit line continuation here. Just use the implicit continuation of
```python
( "blah blah"
  "blah ....")
```